### PR TITLE
Tutorial formatting and clarifications

### PR DESCRIPTION
**Pull request by smith**
When going through the tutorial there were some formatting issues and a few
steps that were not immediately apparent to a Go novice.

Fixes those; hope it will improve the tutorial a tiny bit.

Signed-off-by: Nathan L Smith <smith@nlsmith.com>

#### Description of the changes

Add a detailed description and purpose of your changes.
Link the issue it solves (if there is one).

### Author

- [x] add a risk label after carefully considering the "blast radius" of your changes
- [x] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [ ] assign at least one reviewer

### Reviewer

- [x] review code for readability
- [x] verify that high risk behavior changes are well tested
- [x] check license for any new external dependency
- [x] ask questions about anything that isn't clear and obvious
- [x] approve the PR when you consider it's good to merge

### CODE REVIEW

**Comment by smith** [comment]
Looks like I don't have permissions to assign a reviewer or add labels. I would call it `risk:low`.

**Comment by chaupt** [comment]
Thank you @smith, we'll take a look

File: docs/tutorial.md [hunk 1]
```diff
@@ -41,15 +47,17 @@ $ $GOPATH/bin/nr-integrations-builder --help
...
 Before initializing the integration with `nr-integrations-builder` you have to check that the `govendor` tool (used for managing dependencies) is successfully installed. Run the following command:
 ```bash
 $ govendor
 ```
 You should receive the description about the `govendor` tool with the list of accepted commands. More information about the usage can be found in the [README.md](https://github.com/kardianos/govendor/blob/master/README.md).
 
-**Step 3: Initialize the integration**
+**Step 4: Initialize the integration**
+
+Create and `cd` to the `$GOPATH/company-name-integrations` directory. This is the directory where your integrations will be stored.
```
Comment (areina) [inline docs/tutorial.md]: Probably this `$GOPATH/company-name-integrations` should be the path that you created above: `$GOPATH/src/myorg-integrations/`, right?

File: docs/tutorial.md [hunk 2]
```diff
@@ -543,8 +569,8 @@ $ ./bin/myorg-redis -pretty
 	"metrics": [
 		{
 			"event_type": "MyorgRedisSample",
-            "net.connectionsReceivedPerSecond": 0.5,
-            "query.instantaneousOpsPerSecond": 1
+			""net.connectionsReceivedPerSecond": 0.5,
```
Comment (areina) [inline docs/tutorial.md]: There is an extra double quote here.

File: docs/tutorial.md (hunk 1 above)
Comment (smith) [inline docs/tutorial.md]: I removed this line in 99ce2d3 since we `cd` there two steps above.

File: docs/tutorial.md (hunk 2 above)
Comment (smith) [inline docs/tutorial.md]: Fixed in 99ce2d3.

File: docs/tutorial.md [hunk 3]
```diff
@@ -264,21 +270,37 @@ func populateMetrics(ms *metric.MetricSet) error {
 	if err != nil {
 		return err
 	}
-	splitedLine := strings.Split(string(output), ":")
-	if len(splitedLine) != 2 {
+	splittedLine := strings.Split(string(output), ":")
```
Comment (MagdaPaj) [inline docs/tutorial.md]: This change should be also reflected in the tutorial code: https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/tutorial-code/redis.go

File: docs/tutorial.md [hunk 4]
```diff
@@ -19,7 +19,13 @@ To successfully complete this tutorial you must:
 * Install the [New Relic Infrastructure Agent](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/installation/install-infrastructure-linux)
 
 ## Building the structure of the integration
-**Step1: Install nr-integrations-builder**
+**Step1:** Create the directory where you want to place the integrations (it needs to be under `$GOPATH/src`)
```
Comment (MagdaPaj) [inline docs/tutorial.md]: This Step1 is redundant with the Step 1 from the next section. So we think that it's better to not include it here. 
We've realised that this section has some more redundant elements. We will refactor it to not confuse our users.
Would you agree to remove this part?

File: docs/tutorial.md (hunk 3 above)
Comment (smith) [inline docs/tutorial.md]: Updated in b2095e9.

File: docs/tutorial.md (hunk 4 above)
Comment (smith) [inline docs/tutorial.md]: Agreed. Fixed in b2095e9.

**Comment by smith** [comment]
@MagdaPaj I've added your suggested changes. Thanks for the feedback.

**Comment by MagdaPaj** [comment]
Great Nathan! Thank you!

**Comment by smith** [comment]
@areina I think your items for review have been addressed in subsequent commits.

**Comment by areina** [comment]
Merged! Thank you Nathan for your contribution 👍

**Pull request merged**